platformshconfig/config.py

```diff
@@ -92,7 +92,7 @@ class Config:
     """
     A local copy of all environment variables as of when the object was initialized.
     """
-    _environmentVariables = []
+    _environmentVariables = {}
```

Comment:
Note: that's why type hinting is great; given that we support 3.5+, it would be nice to add some PEP 484 love to this code.